### ISSUE TITLE
hairyhenderson.ca domain has expired!

### ISSUE THREAD

**Issue by docwhat** [hairyhenderson/gomplate#198]
The [hairyhenderson.ca](http://hairyhenderson.ca) domain has expired and all the documentation is unavailable:

> The domain "hairyhenderson.ca" has expired.
>
> If you are the owner, please login to your domain manager at http://domains.phibian.com/ for more details.

**Comment by docwhat** [comment]
It would be cool if there was a docker image with all the documentation on it for local reference. :-)

**Comment by hairyhenderson** [comment]
Thanks @docwhat - been trying to resolve that all day today, but I'm at a cottage so things are going a little bit slower 😉

The domain _was_ renewed, but some bug with the registrar caused it to remain offline. Hopefully it should start resolving within the next ~20mins or so.

**Comment by hairyhenderson** [comment]
@docwhat you can also use https://master--gomplate.netlify.com/ if https://gomplate.hairyhenderson.ca continues to be unavailable

**Comment by hairyhenderson** [comment]
> It would be cool if there was a docker image with all the documentation on it for local reference. :-)

Yup, that _would_ be cool! It's easy to get a local site going with `hugo` - just do:

```console
$ cd docs
$ hugo server --bind 0.0.0.0                          
Started building sites ...
Built site for language en:
0 draft content
0 future content
0 expired content
12 regular pages created
8 other pages created
0 non-page files copied
0 paginator pages created
0 tags created
0 categories created
total in 13 ms
Watching for changes in /Users/dee/gocode/src/github.com/hairyhenderson/gomplate/docs/{content,themes}
Serving pages from memory
Web Server is available at //localhost:1313/ (bind address 0.0.0.0)
Press Ctrl+C to stop
```

So _in theory_ we could just have a `Dockerfile` in the `docs/` directory that installs `hugo` and runs it.

**Comment by hairyhenderson** [comment]
It's back! Closing issue - thanks for reporting @docwhat 🙂
